pyproject.toml

```diff
@@ -23,7 +67,7 @@ known_first_party = [
 ]
 
 [tool.flake8]
-max-line-length = 80
+max-line-length = 88
```

Comment:
My preference is for 80 (two files then fits nicely on my screen  :smile:), but no strong opinion. Change it if you prefer. A bit strange though that black and flake8 are not cooperating on this, since both are configured for 80 in `pyproject.toml`.

Reply:
I set it to 80 for now